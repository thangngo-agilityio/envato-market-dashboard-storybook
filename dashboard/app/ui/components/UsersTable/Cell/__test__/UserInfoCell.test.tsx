import { render } from '@testing-library/react';
import { UserInfoCell } from '..';
import { Table, Tr } from '@chakra-ui/react';

describe('TotalCard component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Table>
        <Tr>
          <UserInfoCell
            name="test"
            imageURL="https://cdn-icons-png.flaticon.com/512/5556/5556468.png"
            email="test@gmail.com"
          />
        </Tr>
      </Table>,
    );

    expect(container).toMatchSnapshot();
  });
});
