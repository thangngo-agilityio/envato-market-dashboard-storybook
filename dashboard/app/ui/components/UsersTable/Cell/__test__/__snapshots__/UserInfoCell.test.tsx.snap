// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalCard component renders correctly 1`] = `
<div>
  <table
    class="chakra-table css-0"
  >
    <tr
      class="css-0"
    >
      <td
        class="css-mzqydp"
      >
        <div
          class="css-1h2sv0j"
        >
          <div
            class="css-10lpwxk"
          >
            <img
              alt="Image of test"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=3840&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fcdn-icons-png.flaticon.com%2F512%2F5556%2F5556468.png&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
            />
          </div>
          <div
            class="css-1rr4qq7"
          >
            <h4
              class="chakra-text css-53tbcl"
            >
              test
            </h4>
            <a
              class="chakra-text css-1axauld"
              href="mailto:test@gmail.com"
            >
              test@gmail.com
            </a>
          </div>
        </div>
      </td>
    </tr>
  </table>
</div>
`;
